sdk/identity/azure-identity-v2/src/main/java/com/azure/identity/v2/AzureCliCredentialBuilder.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.v2;
+
+import com.azure.identity.v2.implementation.models.ClientOptions;
+import com.azure.identity.v2.implementation.models.DevToolsClientOptions;
+import com.azure.identity.v2.implementation.util.IdentityUtil;
+import com.azure.identity.v2.implementation.util.ValidationUtil;
+import io.clientcore.core.instrumentation.logging.ClientLogger;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Fluent credential builder for instantiating a {@link AzureCliCredential}.
+ *
+ * <p>The Azure CLI is a command-line tool that allows users to manage Azure resources from their local machine or
+ * terminal. It allows users to
+ * <a href="https://learn.microsoft.com/cli/azure/authenticate-azure-cli">authenticate interactively</a> as a
+ * user and/or a service principal against
+ * <a href="https://learn.microsoft.com/entra/fundamentals/">Microsoft Entra ID</a>.
+ * The AzureCliCredential authenticates in a development environment and acquires a token on behalf of the
+ * logged-in user or service principal in Azure CLI. It acts as the Azure CLI logged in user or service principal
+ * and executes an Azure CLI command underneath to authenticate the application against Microsoft Entra ID.</p>
+ *
+ * <p><strong>Sample: Construct IntelliJCredential</strong></p>
```

Comment:
```suggestion
 * <p><strong>Sample: Construct AzureCliCredential</strong></p>
```

I opened another PR to fix this in v1